src/hooks/useNodeNavigation.ts

```diff
@@ -32,7 +33,7 @@ function buildPathname(selectedNodePath: UiNode[]) {
 }
 
 function makeDriveNodeUrlComponent(driveNode: UiDriveNode) {
-    const component = driveNode.slug || driveNode.name || driveNode.id;
+    const component = driveNode.slug || driveNode.id || driveNode.name;
```

Comment:
I see we now fall back to the drive ID here, but that would mean we never use the drive name, since the id will always exist. was that intentional?